src/quo2/components/settings/category/reorder/view.cljs

```diff
@@ -20,8 +19,6 @@
   [{:keys [label data blur? theme]}]
   (reagent/with-let [atom-data (reagent/atom data)]
     [rn/view {:style style/container}
-     (when blur?
```

Comment:
sorry if this was already answered, but why are we deleting this blur here? t's wrong or something? 🤔

Reply:
@J-Son89 yes it shouldn't be here. This was added for preview, and we now have blur automatically added in our `preview-container` when needed

Reply:
Nice 👌